src/main/resources/avro/readmethods.avdl

```diff
@@ -72,7 +72,20 @@ record SearchReadsResponse {
 }
 
 /**
-Gets a list of `ReadAlignment` matching the search criteria.
+Gets a list of `ReadAlignment`s for one or more `ReadGroup`s.
+
+`searchReads` operates over a genomic coordinate space of reference sequence
+and position defined by the `Reference`s to which the requested `ReadGroup`s are
+aligned.
+
+If a target positional range is specified, search returns all reads whose
+alignment to the reference genome *overlap* the range. A query which specifies
+only read group IDs yields all reads in those read groups, including unmapped
+reads.
+
+All reads returned (including reads on subsequent pages) are ordered by genomic
+coordinate (by reference sequence, then position). Reads with equivalent genomic
+coordinates are returned in an unspecified but deterministic order.
```

Comment:
I've revised this comment. It was a bit of struggle to find a middle-ground between providing rationale, keeping semantics weak enough to allow flexibility of implementation, and readability. I started to add more detailed rationale, but it felt very out of place. This is developer documentation - I don't think it's appropriate to rationalize all design decisions here unless that information also happens to be useful in writing client code or implementing a server. If someone really cares in this case, they can look at the blame and find my response above.

Hopefully this is also more self-explanatory with the way I've rephrased it. Please take a look.